msgraph-connector-product/products/msgraph-calendar/product.json

```diff
@@ -1,5 +1,27 @@
 {
   "installers": [
+    {
+      "id": "maven-import",
+      "data": {
+        "projects": [
+          {
+            "groupId": "com.axonivy.connector.office365",
+            "artifactId": "msgraph-calendar-demo",
```

Comment:
I'd consider to 'craft' this product.json with a maven-resource-plugin : now in these 3 projects this file is duplicated, and differs only on one line ... can't we create it with maven and inject the current artifactId to install?